sermant-plugins/sermant-service-registry/spring-cloud-registry-plugin/src/main/java/com/huawei/registry/config/RegistryConfigResolver.java

```diff
@@ -66,15 +66,16 @@ private boolean updateConfig(Map<String, Object> dic, DynamicConfigEventType eve
             final String name = field.getName();
             final Object configValue = dic.get(getConfigPrefix() + name);
             if (configValue == null || eventType == DynamicConfigEventType.DELETE) {
-                // 采用默认值覆盖, 默认值会将会改为修改为第一次从配置和环境变量读取覆盖后的配置值
+                // Override with a default value, the default value will be changed to the first time the override is
+                // read from the configuration and environment variables
```

Comment:
The default value will be changed to the value read from the configuration and environment variables for the first time.

Reply:
done